contract/src/main/java/io/neow3j/contract/NeoURI.java

```diff
@@ -74,51 +77,47 @@ public static NeoURI fromURI(String uriString) throws IllegalFormatException {
     }
 
     /**
-     * Builds an Invocation from this NeoURI.
-     * Needs all necessary parameters to create an invocation object.
+     * Builds a transaction builder {@code TransactionBuilder} from this NeoURI.
+     * Needs all necessary parameters to create a transfer invocation.
      *
-     * @return an Invocation object.
+     * @return a TransactionBuilder object.
      */
-    public Invocation.Builder invocationBuilder() throws IOException {
-        if (this.neow3j == null) {
+    public TransactionBuilder transferBuilder() throws IOException {
```

Comment:
The NEP-9, as proposed [here](https://github.com/neo-project/proposals/blob/master/nep-9.mediawiki#Smart_contract_invocation_URI) only specifies NEP-5 transfers.

About using URI for other contract invocations, they wrote the following:
> [...] generic smart contract execution is not necessarily suitable for URI's. This is because both of the security issues, and usability. A client of the URI should have 100% confidence that the URI that they will consume will have its intended effect. [...] If the client wishes to be even more secure, it can whitelist its own set of contracts for each subset of supported smart contract URI's.

So, I will leave the method as is.